packages/model/src/builder.ts

```diff
@@ -100,8 +100,11 @@ class Builder {
     return { _class: core.class.Attribute, type } as unknown as Attribute
   }
 
-  createDocument<M extends Doc> (_class: Ref<Class<M>>, values: Omit<M, keyof Doc>, _id?: Ref<M>): M {
-    const doc = this.memdb.createDocument(_class, values, _id)
+  createDocument<M extends Doc> (_class: Ref<Class<M>>, values: Partial<M>, _id?: Ref<Doc>): M {
+    const doc = this.memdb.createDocument<M>(_class, values)
+    if (_id) {
+      doc._id = _id
+    }
```

Comment:
Again, same: why remove optional id from `memdb.createDocument` and do this tricks?

Reply:
Will revert this change.